core/store/migrations/0036_create_csa_keys.go

```diff
@@ -0,0 +1,31 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up36 = `
+CREATE TABLE csa_keys(
```

Comment:
This is a temporary table, yes?

Reply:
Yes. Once the keystore refactoring has been complete, this will be merged into a single keys table